pkgs/development/python-modules/libcst/default.nix

```diff
@@ -15,11 +15,11 @@
 
 buildPythonPackage rec {
   pname = "libcst";
-  version = "0.3.20";
+  version = "0.3.21";
+  format = "setuptools";
+
   disabled = pythonOlder "3.6";
 
-  # Some files for tests missing from PyPi
-  # https://github.com/Instagram/LibCST/issues/331
```

Comment:
We fetch many packages from GitHub instead of pypi. It doesn't really matter from where we fetch the package and switching that back wouldn't really do anything IMO.